routes/index/index.marko

```diff
@@ -0,0 +1,20 @@
+<html>
```

Comment:
I didn't make it clear in the tutorial, but having an `index.marko` in the route of you project (e.g. `/marko-color-picker/index.marko`), will automatically get served as the index route's template. The tutorial does not reflect that. Perhaps you could update the tutorial to describe this? I think it's important to note that adding a `routes/index/index.marko` or `routes/some-route/index.marko` yields routes and that `marko-starter` automatically resolves routes from the `routes` folder, but ultimately, we should keep the `index.marko` at the route of the project. The Marko try-online looks for an `index.marko` at the root of each virtual project (e.g. [color-picker try-online](http://markojs.com/try-online/?file=%2Fmarko-color-picker%2Findex.marko&gist=)), so we should leave it there. Can you update the tutorial to describe this and remove this file? Thanks!